libc/utils/gpu/loader/amdgpu/Loader.cpp

```diff
@@ -34,6 +34,19 @@
 #include <tuple>
 #include <utility>
 
+// The implicit arguments of COV5 AMDGPU kernels.
```

Comment:
Is there a way to share this structure between here and `openmp/libomptarget/plugins-nextgen/amdgpu/utils/UtilitiesRTL.h`?

Otherwise, someone will have to keep updating this whenever there are changes in the AMDGPU plugin (like #65325 )

Reply:
Yeah that's a future endeavor. Probably something that could be solved by having the offloading  directory for a lot of these HSA wrappers. For now it's easier to keep them separate.